### DISCUSSION TITLE
Issues with newer versions of Angular

### DISCUSSION THREAD

**Discussion by lperezq6cyber** [timostamm/protobuf-ts#624] [General]
Hello everyone, I'm having this issue `Error: Unexpected value 'PbDatePipeModule' imported by the module 'SharedModule'. Please add an @NgModule annotation.` after upgrading to Angular 16.

I thought maybe I could fork the project and see what modifications could be done on `@protobuf-ts/runtime-angular`, but to my surprise that package doesn't exist anymore in the library repo.

Any idea?

Thanks!!

**Comment by jcready** [comment]
Angular support was dropped 9 months ago https://github.com/timostamm/protobuf-ts/pull/522
